src/metalnx-web/src/main/resources/views/collections/treeViewForUserForm.html

```diff
@@ -160,23 +150,10 @@
 		$(document).ready(function(){
 			$("select option:selected").each(function() {
 				if ( $(this).val() == "none" ) {
-					$(this).parents("tr").find("input[type='checkbox'].userBookmarkCheckbox").prop('checked', false);
-					$(this).parents("tr").find("input[type='checkbox'].userBookmarkCheckbox").prop("disabled", true);
-				}
```

Comment:
We're missing a closing curly brace for the if-stmt, but it also seems like the if-stmt isn't needed anymore.

Reply:
It would also appear that the surrounding function would become empty if the `if` statement is removed.